ui-v2/tests/unit/helpers/format-number-call-if-type-test.js

```diff
@@ -0,0 +1,13 @@
+import { callIfType } from 'consul-ui/helpers/format-number';
+import { module, test } from 'qunit';
+
+module('Unit | helper | format-number.callIfType');
+const formatNumber = callIfType('number')(function() {
+  return true;
+});
+test('it returns the argument passed if not a number', function(assert) {
+  assert.equal(formatNumber(['1,000']), '1,000');
+});
+test("it returns the argument to locale string if it's a number", function(assert) {
```

Comment:
This test name is inaccurate. Should be something like "it passes the first param through the provided function if it's a number".

Reply:
Hmm yeah, I know what you mean but actual usage of this is:

```
{{format-number "1,000"}}
```

'The first argument' if you are talking about helpers is the first element in an array. This is what I mean about being on the fence about what to do with this 'helper helper' or 'helper util'. Right now it's specifically a util/helper that is only used with helpers.

You're right though, I do think it could use more explanation. I think I might move all this into `utils/helpers` (both in the app and tests). Then add maybe a README in that folder explaining that everything in that folder is meant for helpers/uses helper signatures/uses helper terminology.

I've added something more into the test name for now, plus some explanation via inline comments seeing as its only a single file for the moment.

I actually saw @meirish had something similar to this specific 'helper helper' in vault so it's definitely an idea I might expand upon.

Let me think about this more. What do you think of the above suggestion? Shout me if you want to chat offline.

Reply:
Yeah, that folder structure makes sense to me 👍